river/command/layout.zig

```diff
@@ -83,3 +84,18 @@ pub fn sendLayoutCmd(
     layout.layout.sendUserCommand(args[2]);
     if (layout == output.layout) server.root.applyPending();
 }
+
+/// Called by the layout manager to update the number of Views in the main area.
```

Comment:
This command may also be called by the user directly.
The concept of main count should not exist in river, only the layout generator.
This should be folded into the `attach-mode` command:
```sh
riverctl attach-mode aside 2
```